app/models/service_load_balancer.rb

```diff
@@ -0,0 +1,68 @@
+class ServiceLoadBalancer < Service
+  include ServiceLoadBalancerMixin
+  include ServiceLoadBalancerOptionsMixin
```

Comment:
ok, will do, might be goog to rename that one, since this has nothing to do with orchestration